enhancements/console/dynamic-plugins.md

```diff
@@ -534,6 +602,26 @@ Both `v1` and `v1alpha1` version are supported. `v1alpha1` plugins will get
 converted by the conversion webhook server into `v1` representation.
 Conversion webhook server is part of the `console-operator` pod.
 
+#### Content Security Policy
+For Content Security Policy (CSP) feature to be considered stable:
+* Update the OpenShift official documentation to include detailed guidelines
+on configuring `ConsolePluginCSP` in the `ConsoleDynamicPlugin` CRD, along with
+recommendations.
+* Add CSP feature to release notes.
```

Comment:
We should document clearly when this will be enforced in the release notes.

Reply:
added at the end of the changes:
```md
To ensure adequate monitoring and adjustment, Console will continue to operate
in `Content-Security-Policy-Report-Only` mode for at least one release after
the above requests are met. After this period, Console will switch to using the
`Content-Security-Policy` header, enabling full enforcement of CSP policies.
```